lib/ui/cachedblock.js

```diff
@@ -0,0 +1,55 @@
+const Block = require('./block');
```

Comment:
With Webpack and Babel now installed, you can use
```JavaScript
import Block from './block';
import { blockSize } from './variables';
```
instead which is the ES6 way of doing things.